[2024-06-03T22:21:02.100+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:21:02.115+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-06-03T22:21:02.121+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-06-03T22:21:02.121+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:21:02.130+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-11-01 00:00:00+00:00
[2024-06-03T22:21:02.137+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=29940) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:21:02.138+0000] {standard_task_runner.py:63} INFO - Started process 29942 to run task
[2024-06-03T22:21:02.138+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-11-01T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpylvsbyqe']
[2024-06-03T22:21:02.139+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask blunders_etl
[2024-06-03T22:21:02.180+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:21:02.253+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-01T00:00:00+00:00'
[2024-06-03T22:21:02.254+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:21:02.255+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:21:02.261+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:24:35.606+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:24:35.607+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:24:35.615+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-11-01T00:00:00+00:00, execution_date=20211101T000000, start_date=20240603T222102, end_date=20240603T222435
[2024-06-03T22:24:35.639+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:24:35.652+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:24:35.654+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:46:50.954+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:46:50.971+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-06-08T19:46:50.977+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-06-08T19:46:50.977+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:46:50.989+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-11-01 00:00:00+00:00
[2024-06-08T19:46:50.996+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=30377) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:46:50.998+0000] {standard_task_runner.py:63} INFO - Started process 30379 to run task
[2024-06-08T19:46:50.998+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-11-01T00:00:00+00:00', '--job-id', '309', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpx2ls8_gh']
[2024-06-08T19:46:50.999+0000] {standard_task_runner.py:91} INFO - Job 309: Subtask blunders_etl
[2024-06-08T19:46:51.033+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:46:51.096+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-01T00:00:00+00:00'
[2024-06-08T19:46:51.097+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:46:51.098+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:46:51.105+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T19:50:11.955+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T19:50:11.955+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T19:50:11.963+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-11-01T00:00:00+00:00, execution_date=20211101T000000, start_date=20240608T194650, end_date=20240608T195011
[2024-06-08T19:50:12.015+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T19:50:12.028+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T19:50:12.030+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:45:09.193+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:45:09.207+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-07-08T01:45:09.213+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [queued]>
[2024-07-08T01:45:09.213+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:45:09.222+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2021-11-01 00:00:00+00:00
[2024-07-08T01:45:09.229+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31246) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:45:09.231+0000] {standard_task_runner.py:63} INFO - Started process 31248 to run task
[2024-07-08T01:45:09.231+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2021-11-01T00:00:00+00:00', '--job-id', '693', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp0wvl5_m5']
[2024-07-08T01:45:09.232+0000] {standard_task_runner.py:91} INFO - Job 693: Subtask blunders_etl
[2024-07-08T01:45:09.268+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2021-11-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:45:09.328+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2021-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-01T00:00:00+00:00'
[2024-07-08T01:45:09.330+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:45:09.331+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:45:09.337+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T01:48:27.877+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T01:48:27.877+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T01:48:27.885+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2021-11-01T00:00:00+00:00, execution_date=20211101T000000, start_date=20240708T014509, end_date=20240708T014827
[2024-07-08T01:48:27.923+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T01:48:27.941+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T01:48:27.942+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
